Riot/Modules/Common/Activities/ActivityRequest.swift

```diff
@@ -0,0 +1,25 @@
+// 
+// Copyright 2021 New Vector Ltd
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import Foundation
+
+/// A request used to create an underlying `Activity`, allowing clients to only specify the visual aspects of an activity.
+struct ActivityRequest {
```

Comment:
Can we have something like `displayAfter` parameter?
It will help to hide some activity spinners if the underlying operation takes less than 1 second (or less, this is TBD)

Reply:
If it is happening for that short, wouldn't the `dismiss` just be called right away, thus not showing any UI? When testing this in the application I did not see any spinners that were flickering but maybe I wasn't finding the right use cases.